### PR TITLE
sdk v2.0.0

### PR DESCRIPTION
**Pull request by CleanBread**

### CODE REVIEW

**Review by Avvrik — COMMENTED** [review]
added comments to fix grammar and simplify the language.

File: pages/guides/sdk/data-hooks/useGames.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Callout } from 'components'
 
-Hook is used for fetch pre-match games.
+Hook is used for fetch pre-match and live games.
```
[the remark this answers is not in the document]
Comment (CleanBread) [reply pages/guides/sdk/data-hooks/useGames.mdx]: e3bc8419a776f09d0fa51bd55b694cf511254e9e

File: pages/guides/sdk/data-hooks/useConditions.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { Callout } from 'components'
 
-Hook is used for fetch the [`Conditions`](/concepts/glossary#condition) of specific game.
+Hook is used for fetch pre-match and live [`Conditions`](/concepts/glossary#condition) of specific game.
```
[the remark this answers is not in the document]
Comment (CleanBread) [reply pages/guides/sdk/data-hooks/useConditions.mdx]: e3bc8419a776f09d0fa51bd55b694cf511254e9e

File: pages/guides/sdk/setup.mdx [hunk 3]
```diff
@@ -21,75 +19,21 @@ function Providers(props: { children: React.ReactNode }) {
...
-  useWatchers()
-
-  return null
-}
-```
-
-After completing the watchers setup, you can begin using the [Watch Hooks](/guides/sdk/watch-hooks/useOutcome).
-
-
+// ATTN: need to rewrite @Avvrik
```
[the remark this answers is not in the document]
Comment (CleanBread) [reply pages/guides/sdk/setup.mdx]: 40203e3c9af61d0f26face3296d8673302608c28

File: pages/guides/sdk/write-hooks/usePrepareBet.mdx [hunk 4]
```diff
@@ -56,9 +69,11 @@ const {...} = usePrepareBet({
 
 ```ts
 {
-  amount: string // bet amount
-  slippage: number // 5%
+  betAmount: string // bet amount
+  slippage: number // in percents
```
[the remark this answers is not in the document]
Comment (CleanBread) [reply pages/guides/sdk/write-hooks/usePrepareBet.mdx]: 66c59282e1d2cb0962f6686c7ccba2d85705dbfb

File: pages/guides/sdk/providers/socket.mdx [hunk 5]
```diff
@@ -0,0 +1,25 @@
+The `SocketProvider` facilitates the subscription functionality for real-time market updates during live games, specifically designed for Watch Hooks.
```
[the remark this answers is not in the document]
Comment (CleanBread) [reply pages/guides/sdk/providers/socket.mdx]: 1d0d4a2a01367028692ffc9a143d49c5961b2f4c

File: pages/guides/sdk/providers/chain.mdx [hunk 6]
```diff
@@ -39,6 +39,38 @@ import { useChain } from '@azuro-org/sdk'
 const { appChain, walletChain, contracts, betToken, isRightNetwork, setAppChainId } = useChain()
 ```
 
+<Callout type="info">
+The `ChainProvider` stores the current chain ID in a cookie under the key `appChainId` to restore `appChain`, for example, after page refresh.
```
[the remark this answers is not in the document]
Comment (CleanBread) [reply pages/guides/sdk/providers/chain.mdx]: 6bc6d4690507fdf1d667c3bb83e914915126f2df

File: pages/guides/sdk/providers/apollo.mdx [hunk 7]
```diff
@@ -36,5 +30,33 @@ function Providers(props: { children: React.ReactNode }) {
...
+```
+
+#### Return Value
+
+```ts
+import { ApolloClient, type NormalizedCacheObject } from '@apollo/client'
+
+{
+  prematchClient: ApolloClient<NormalizedCacheObject> | null // used for get pre-match data
+  liveClient: ApolloClient<NormalizedCacheObject> | null // used for get live data
```
[the remark this answers is not in the document]
Comment (CleanBread) [reply pages/guides/sdk/providers/apollo.mdx]: 6bc6d4690507fdf1d667c3bb83e914915126f2df

File: pages/guides/sdk/providers/apollo.mdx [hunk 8]
```diff
@@ -36,5 +30,33 @@ function Providers(props: { children: React.ReactNode }) {
...
+}
+```
+
+#### Return Value
+
+```ts
+import { ApolloClient, type NormalizedCacheObject } from '@apollo/client'
+
+{
+  prematchClient: ApolloClient<NormalizedCacheObject> | null // used for get pre-match data
```
[the remark this answers is not in the document]
Comment (CleanBread) [reply pages/guides/sdk/providers/apollo.mdx]: 6bc6d4690507fdf1d667c3bb83e914915126f2df

File: pages/guides/sdk/data-hooks/usePrematchBets.mdx [hunk 9]
```diff
@@ -0,0 +1,98 @@
+import { Callout } from 'components'
+
+Hook is used for fetch pre-match bets history for specific bettor.
```
[the remark this answers is not in the document]
Comment (CleanBread) [reply pages/guides/sdk/data-hooks/usePrematchBets.mdx]: 6bc6d4690507fdf1d667c3bb83e914915126f2df

**Pull request closed without merging**